R/pkg/inst/tests/test_sparkSQL.R

```diff
@@ -359,10 +359,21 @@ test_that("Collect DataFrame with complex types", {
   expect_equal(bob$height, 176.5)
 })
 
-test_that("jsonFile() on a local file returns a DataFrame", {
-  df <- jsonFile(sqlContext, jsonPath)
+test_that("read.json() on a local file returns a DataFrame", {
```

Comment:
although jsonFile() is deprecated, we'd better to keep the test cases for it until it is removed?